stubs/stripe/stripe/api_resources/account.pyi

```diff
@@ -21,3 +21,53 @@ class Account(CreateableAPIResource, DeletableAPIResource, ListableAPIResource):
     def persons(self, **params): ...
     def deauthorize(self, **params): ...
     def serialize(self, previous): ...
+    @classmethod
+    def capabilitys_url(cls, id, nested_id=...): ...
+    @classmethod
+    def capabilitys_request(
```

Comment:
this looks like a typo, but is apparently due to a bug [here](https://github.com/stripe/stripe-python/blob/053f1a2c376dd40b723f680bfc48c624be25b5c6/stripe/api_resources/abstract/nested_resource_class_methods.py#L50) and [here](https://github.com/stripe/stripe-python/blob/053f1a2c376dd40b723f680bfc48c624be25b5c6/stripe/api_resources/abstract/nested_resource_class_methods.py#L28)